examples/experimental/dagster-airlift/scripts/mirror_tutorial.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
```

Comment:
If these are copy pasted from elsewhere can you indicate from where so we can consolidate at some point?

Reply:
Not quite copy pasted, but yea I think some consolidation is in order

Reply:
"Inspired by"

Reply:
😜